bids/analysis/transformations/compute.py

```diff
@@ -49,11 +54,18 @@ def _transform(self, var, model='spm', derivative=False, dispersion=False,
         elif model != 'fir':
             raise ValueError("Model must be one of 'spm', 'glover', or 'fir'.")
 
-        convolved = hrf.compute_regressor(vals, model, onsets,
-                                          fir_delays=fir_delays, min_onset=0)
+        min_interval = min(np.ediff1d(np.sort(var.onset)).min(),
```

Comment:
Maybe add a comment here for posterity explaining `min_interval`—it will reduce the maintenance burden a year or two down the line